package.json

```diff
@@ -162,24 +161,25 @@
     "mini-css-extract-plugin": "2.9.2",
     "mocha": "10.7.3",
     "moment": "^2.29.4",
-    "ng-packagr": "17.1.2",
+    "ng-packagr": "19.2.2",
     "nock": "13.5.5",
     "npm-run-all": "^4.1.5",
-    "nx": "^20.0.0",
+    "nx": "20.8.0",
     "postcss": "8.4.41",
     "postcss-sass": "^0.5.0",
     "prettier": "2.8.8",
     "rimraf": "6.0.1",
-    "sass-loader": "16.0.4",
+    "sass-loader": "16.0.5",
     "shelljs": "^0.8.5",
     "spdx-license-list": "^6.9.0",
     "storybook": "^8.2.9",
     "stylelint": "16.8.2",
     "stylelint-config-standard-scss": "^13.1.0",
     "ts-node": "^10.9.2",
     "tsconfig-paths": "^4.1.1",
-    "typescript": "5.3.3",
-    "webpack": "5.97.1",
+    "typescript": "5.7.3",
```

Comment:
based on compatibility documentation we can use even higher ts version (we can increase it to 5.8.x - any version before 5.9.x) because Angular is upgraded to 19.2.x 
https://angular.dev/reference/versions#actively-supported-versions